### PR TITLE
ADD: School behavior is now visualizable using a pie chart or a bar g…

### PR DESCRIPTION
**Pull request by davidabrahams**
…raph! Also, the bar graph y axis now dynamically names itself according to the data selected. Before, it was always named Stars

### CODE REVIEW

**Comment by davidabrahams** [comment]
Fixed #174

File: public/javascripts/d3Controller.js [hunk 1]
```diff
@@ -185,67 +172,88 @@ var d3Controller = function($scope, $http, $state) {
...
   $http.get('/api/student/dataList/' + $scope.currentStudent._id).then(function successCallback(
     response) {
 
     // iterate over the forms in the DB
     response.data.forEach(function(entry) {
       $scope.dates.push(entry.date);
+      // put school behavior in the current dictionary
+      for (var key in entry['schoolBehavior']) {
+        if (entry['schoolBehavior'].hasOwnProperty(key)) {
+          entry[key] = entry['schoolBehavior'][key]
```
Comment (nmohamed) [inline public/javascripts/d3Controller.js:252]: ;

**Comment by nmohamed** [comment]
:palm_tree: :bikini: :sun_with_face: :watermelon:

**Comment by shrutiyer** [comment]
:+1:

**Comment by aconsilvio** [comment]
The pie/bar charts are only showing true entries, which I don't think is what we want.  This should be fixed before we merge

**Comment by davidabrahams** [comment]
Is it possible instead of having "false" on the DB we have "undefined",
which is getting filtered?

**Comment by aconsilvio** [comment]
From what I understand they just aren't set at all when we don't use them.  Like it creates objects {Suspension:true} where anything not checked just doesn't exist, so maybe?

**Comment by davidabrahams** [comment]
So yeah, my comment above is what's happening, The fix is just to initiate
them to false client side

**Comment by BrennaManning** [comment]
:+1:

**Pull request merged**